contrib/test_jenkins.sh

```diff
@@ -58,6 +58,81 @@ echo "Build without IB verbs"
 ../contrib/configure-release --without-verbs
 make $make_opt
 
+if [ -n "$JENKINS_PPC_SERVER" ]; then
```

Comment:
1st of all, I want stabilize the infra problems, benchmark and modules.
You will see false alarms with old code, and when all components of infra will be fixed, I'll fix code.

10x